plugins/searchdocs.py

```diff
@@ -30,7 +30,7 @@ def search(self, msg, arg):
             return (self.API_DOCS +
                     '/search.html?q=' +
                     '+'.join(re.split(r'\s+', search_string)))
-        elif doc_type.lower() == 'user':
+        else:
```

Comment:
No need for else IMO when this will be the last condition instead directly return

Reply:
My own words thrown back at me ? ;)